deploy-manage/cloud-organization/billing/manage-billing-notifications.md

```diff
@@ -0,0 +1,64 @@
+---
+navigation_title: Manage notifications
+applies_to:
+  deployment:
+    ess: ga
+  serverless: ga
+products:
+  - id: cloud-hosted
+  - id: cloud-serverless
+---
+
+# Manage usage and cost notifications [billing-notifications]
+
+To help you to better understand your costs and manage spending on {{ecloud}}, you can configure email alerts to be sent when your monthly usage reaches a specified threshold, relative to a budget that you define for your account. You can also opt to receive a biweekly summary of your organization's usage and estimated costs for the previous month.
+
+When configured, any email notifications are sent to users who are members of the Organization owner or Billing admin [user role](/deploy-manage/users-roles/cloud-organization/user-roles.md#ec_organization_level_roles).
+
+To configure email notifications for your {{ecloud}} billing:
+
+1. Log in to the [{{ecloud}} Console](https://cloud.elastic.co?page=docs&placement=docs-body).
+2. From a deployment or project on the home page, select **Manage**.
+3. From the lower navigation menu, select **Billing and Subscription**.
+4. Follow the steps to set up or change the types of email notifications you'd like to receive:
+    * [Configure budget emails](#configure-budget-emails)
+    * [Configure usage summmary emails](#configure-usage-summary-emails)
+
+Note that [Credit consumption emails](#configure-credit-consumption-emails) are also shown on the **Billing and Subscription** page, but these notifications are not configurable.
+
+## Configure budget emails [configure-budget-emails]
+
+Budget alerts can be sent when your organization's total monthly usage exceeds a threshold defined in a configurable budget.
+
+To create a budget and configure budget emails:
+
+1. Open the **Notifications** page.
```

Comment:
can we change the name from `Notifications` to `Budgets and notifications`  (and all uses in different places) ? cc @briannebartlett @kyrspl